### ISSUE TITLE
Add range support for AES/CBC

### ISSUE THREAD

**Issue by dekobon** [joyent/java-manta#154]

**Comment by cburroughs** [comment]
Is this (and #155) excluded from the 3.0 milestone intentionally? (Is CTR-only a minimal viable product?)

**Comment by dekobon** [comment]
Yes. AES/CTR is the MVP. I poked around with AES/CBC. I wanted to verify if we needed per cipher mode plaintext to ciphertext translation implementations and it turns out that the answer is yes. So, AES/CBC would be a good one to finish post the 3.0 milestone.
